climate.py

```diff
@@ -75,6 +81,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
 class AwesomeHeater(ClimateDevice):
     """Representation of a demo climate device."""
+    _current_mode = HVAC_MODE_AUTO
```

Comment:
```suggestion
```
Should be moved inside __init__ and with 'self.'

Reply:
OK. I made it a overrideable class member, but it could just as well be a instance member

Reply:
I must admit I had to google the difference, but since you can override a single zone I guess it makes more sense to have it as an instance member?